src/source.hpp

```diff
@@ -22,4 +22,67 @@ namespace source {
     static_assert(offsetof(GameLumpHeader, version) == 0x6);
     static_assert(offsetof(GameLumpHeader, offset)  == 0x8);
     static_assert(offsetof(GameLumpHeader, length)  == 0xC);
+
```

Comment:
None of these structs are in the Source Engine
they should all be in `titanfall.hpp`